### PR TITLE
fix: user profile area should be clickable #2566

### LINKED ISSUE #2566: Entire user profile area should be clickable, in bottom left
Right now it's sort of fiddly to open your user profile because, in the bottom left of the sidebar, only the username and avatar is clickable. 

The whole area should be clickable, and the hover state should change to reflect this.

### PR DESCRIPTION
**Pull request by okrayum**
This PR addresses issue #2566 by ensuring that the entire user profile area in the sidebar is clickable.

- Moved padding from the parent container to the button to ensure that the button area is fully clickable while maintaining the same size and style.
- Updated the hover state to apply to the entire button area.

This shows the previous next to after the changes I made:

https://github.com/user-attachments/assets/c805b034-5f3d-4675-bfe0-07a288f7eb7f

### Pull Request Checklist

- [x] I have linked this PR to a related GitHub issue.
- [x] I have added a description of the change (and Github issue number, if any) to the root [CHANGELOG.md](
https://github.com/user-attachments/assets/fce356ef-e62d-441b-971c-9797c823429e).

### (Optional) Mobile checklist

Please ensure you completed the following checks if you did any changes to the mobile package:

- [ ] I have run e2e tests for mobile
- [ ] I have updated base screenshots for visual regression tests

This PR closes #2566

### CODE REVIEW

**Comment by holmesworcester** [comment]
Thank you! I just approved tests to run. It might be necessary to fix e2e or snapshot tests, but possibly not!

**Comment by okrayum** [comment]
@holmesworcester , ok thank you! I already see some checks failed.

**Comment by holmesworcester** [comment]
Will you have a look and fix them? It should be straightforward and please let us know if not!

Thanks again for the initial contribution ❤️

**Comment by okrayum** [comment]
Yes, I will check them now. Thanks!
@holmesworcester  , I see the first failure as simple, the others I may need some guidance on if possible.
Thanks!

**Comment by okrayum** [comment]
I have addressed previous failing checks, let me know please.
Thanks!

**Comment by okrayum** [comment]
I'm unsure where I am going wrong with the failing checks. This is what I have for the current 4 that are showing as failing.

1. Check / check (ubuntu-20.04) (pull_request) , I have made sure the semicolon is no longer there.
Screenshot 2024-08-12 154157
Screenshot 2024-08-12 152101

2. Dependency Review / dependency-review (pull_request), this was not changed.

3. Desktop scroll regression tests / regression-tests (ubuntu-20.04) (pull_request), the tests now should pass.
Screenshot 2024-08-12 152025

4. Desktop tests / desktop-tests (ubuntu-20.04) (pull_request), the tests now should pass.
Screenshot 2024-08-12 153509

**Comment by holmesworcester** [comment]
Sorry for the friction here and thanks for looking into it. @ikoenigsknecht will look at this today!

**Comment by okrayum** [comment]
Sounds good thank you very much. It seems like my last commit didn't take any of the staged changes. They are showing now, so I'm anxious to see how they do with the checks. I appreciate it!

**Comment by okrayum** [comment]
Hello  @ikoenigsknecht , let me know what i need to fix.
Thanks!

**Comment by okrayum** [comment]
Hello @ikoenigsknecht, I was just curious if there was any feedback after my last fixes of the failing checks. I understand everyone is busy, so just checking in.
Thank you!

File: packages/desktop/libssl.so
Comment (adrastaea) [inline packages/desktop/libssl.so:1]: @okrayum Is there a reason this file needed to be included here? We already have this in the `3rd-party/tor/` directory.

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "npm": "8.19.2"
   },
   "devDependencies": {
+    "braces": "^3.0.3",
```
Comment (adrastaea) [inline package.json]: I don't think we use this anywhere. Why was it added?

**Review by adrastaea — CHANGES_REQUESTED** [review]
Looks good other than some seeming unrelated inclusions. I like the effect! Will approve and merge once those questions are resolved.

File: package.json (hunk 1 above)
Comment (okrayum) [reply package.json]: I had a previous failure in the Dependency review and it noted the "braces" package, I believe it was updated. 
Running the failing checks locally, the only one that continues to fail is "npm run test backwardsCompatibility.test.ts" which hangs when it is supposed to copy the invitation code to the clipboard.

File: packages/desktop/libssl.so
Comment (okrayum) [reply packages/desktop/libssl.so:1]: I have used the following steps listed in the guidelines:
`npm i lerna@6.6.2`
`npm i typescript@4.9.3`
`npm run lerna bootstrap`

I didn't add it manually.
Could running npm install in that directory have added it?

File: package.json (hunk 1 above)
Comment (okrayum) [reply package.json]: @adrastaea , As I don't see "braces" in the package.json in the repo I forked from, I have removed it. Should I push my changes to this commit?

File: packages/desktop/libssl.so
Comment (adrastaea) [reply packages/desktop/libssl.so:1]: Could you remove it please?

File: package.json (hunk 1 above)
Comment (adrastaea) [reply package.json]: Yes please

Comment (adrastaea) [reply package.json]: Also, backwardsCompatibility.test.ts is failing on our base branch so no need to worry about that

Comment (okrayum) [reply package.json]: Ok, @adrastaea . I have removed `braces` and `libssl.so`  and addressed failing tests that I could.
Thanks!

File: package.json [hunk 2]
```diff
@@ -21,5 +21,8 @@
     "husky": "^9.0.11",
     "lerna": "^6.6.2",
     "typescript": "^4.9.3"
+  },
+  "dependencies": {
+    "micromatch": "^4.0.8"
```
Comment (adrastaea) [inline package.json]: Sorry to keep going back and forth on this, but could you remove this dependency?

**Review by adrastaea — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (okrayum) [reply package.json]: Absolutely, it was also a Dependency review failure, it must have been associated with `braces`

**Comment by okrayum** [comment]
@adrastaea , Thank for all of your help!

**Pull request merged**